src/Compilers/Shared/GlobalAssemblyCacheHelpers/GlobalAssemblyCache.cs

```diff
@@ -27,7 +25,9 @@ private static GlobalAssemblyCache CreateInstance()
             }
             else
             {
-                return new ClrGlobalAssemblyCache();
+                return System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription.Contains(".NET Framework")
```

Comment:
if there is a preferred way to do this let me know.

Reply:
Believe this is the preferred way. Pattern occurs several places in our code base.